setup/iodbc.ini

```diff
@@ -0,0 +1,10 @@
+[ODBC Data Sources]
+ADF_Test = "MongoDB Atlas SQL ODBC Driver"
+
+[ADF_Test]
+Driver = %DRIVER_LIB_PATH%/libatsql.dylib
+Database = %ADF_TEST_DB%
+User = %ADF_TEST_USER%
+Password = %ADF_TEST_PWD%
+Uri = mongodb://localhost:27017
```

Comment:
Should we change it to be able to use an `ADF_TEST_HOST` variable? Just in case we have to use a remote test instance someday? It seems that the Windows setup uses `ADF_TEST_HOST` but in a funky way. It replace it with this URI. So, maybe what we actually want to do is clean-up the Windows setup and set the uri directly in the Windows setup file.

Reply:
I thought about this, and wavered back and forth I'll add one.

Reply:
Added